InnerEye/ML/model_training.py

```diff
@@ -288,7 +271,7 @@ def model_train(checkpoint_path: Optional[Path],
         sys.exit()
 
     logging.info("Choosing the best checkpoint and removing redundant files.")
```

Comment:
It might be clearer for the user to say something like "Removing redundant checkpoint files. Retaining the most recent" here?